sdk/search/Azure.ResourceManager.Search/api/Azure.ResourceManager.Search.netstandard2.0.cs

```diff
@@ -153,6 +156,56 @@ public partial class SharedSearchServicePrivateLinkResourceData : Azure.Resource
 }
 namespace Azure.ResourceManager.Search.Models
 {
+    public enum AadAuthFailureMode
+    {
+        Http403 = 0,
+        Http401WithBearerChallenge = 1,
+    }
+    public partial class DataPlaneAuthOptions
+    {
+        public DataPlaneAuthOptions() { }
+        public Azure.ResourceManager.Search.Models.AadAuthFailureMode? AadAuthFailureMode { get { throw null; } set { } }
+        public System.BinaryData ApiKeyOnly { get { throw null; } set { } }
+    }
+    public partial class EncryptionWithCmk
```

Comment:
This name is quite generic, could we rename the name of this class to `SearchEncryptionWithCmk` to specify?